services/utils/objectToLabelValueArray.js

```diff
@@ -0,0 +1,5 @@
+export const objectToLabelValueArray = (object) => {
+    return Object.entries(object).map(([key, value]) => {
+        return {label: value, value: key}
+    });
+}
```

Comment:
GitHub highlights with red a blank line is required 

```suggestion
}

```